routes/commentRoutes.js

```diff
@@ -1,5 +1,26 @@
 const express = require('express');
+const commentController = require('../controllers/commentController');
+const authMiddleware = require('../middlewares/userAuth');
 
 const router = express.Router();
 
+router.post(
+  '/:slug',
+  authMiddleware.isAuthenticated,
+  authMiddleware.isAuthorized,
```

Comment:
probably won't need authorized? just authenticated should be sufficient for comment posting

Reply:
OK, I agree, have changed.